soc/arm/nxp_lpc/lpc55xxx/soc.c

```diff
@@ -62,6 +62,8 @@ static ALWAYS_INLINE void clock_init(void)
 	defined(CONFIG_SOC_LPC55S28) || defined(CONFIG_SOC_LPC55S36) || \
 	defined(CONFIG_SOC_LPC55S69_CPU0)
     /*!< Set up the clock sources */
+	/*!< Power Management Controller initialization */
```

Comment:
I assumed it would help for maintainability, 
I notice the comments underneath this line also had the Doxygen
style comments, e.g. the *Ensure FRO is on, underneath the comment,
*Configure FRO192M,
so I assumed this comment might need it to?
I can take this off if its not needed.

Reply:
I guess it's ok if there are others in this file, but honestly you should clean this up at some point @danieldegrasse or @mmahadevan108.